### ISSUE TITLE
Why do we need to wrap tests in main() function?

### ISSUE THREAD

**Issue by masimakopoulos** [lathonez/clicker#65]
Hi and thanks for the great guide.

Not really an issue as much as a question. Is there a particular reason why we need to wrap tests in main() function? 

test-main.js
```
        if (module.hasOwnProperty('main')) {
          module.main();
        } else {
          console.log('MODULE: ' + path);
          throw new Error('Module ' + path + ' does not implement main() method.');
        }
```

It just feels weird.

**Comment by lathonez** [comment]
Hey,

Apologies in the delay getting back to you, I'm travelling atm.

This project is basically a fork of [Angular 2 Seed](https://github.com/mgechev/angular2-seed/blob/master/test-main.js#L49-L53)

I had always assumed this came from Karma, but I cant find a reference to it anywhere other than Angular 2 Seed. After a solid half hour of googling I can't figure it out.

Have asked in [gitter](https://gitter.im/mgechev/angular2-seed) hopefully someone will get back to me.

Cheers,

**Comment by lathonez** [comment]
The maintainer @mgechev got back to me:

> @lathonez it comes from the angular's test-main.js https://github.com/angular/angular/blob/master/test-main.js#L32-L40

The logic (invoking main) is in the [very first commit](https://github.com/angular/angular/commit/c3b442ea53ad43fd348f80a579a82060aa239c7b).

I appreciate I haven't really answered your question. I guess you'll need to raise an issue against angular  (or maybe you can figure it out from the history, I can't).

I try to keep this project in line with Angular 2 Seed as much as possible as that project is very heavily maintained and it makes my life easier porting new features over.

If you get an answer let me know.

**Comment by lathonez** [comment]
Hey

As per #68 getting merged back (440ac08), we no longer need a main method.

Cheers

**Comment by lathonez** [comment]
#191
